### DISCUSSION TITLE
Is there any known/expected errors if using abris on Spark 3.3.0 ?

### DISCUSSION THREAD

**Discussion by EstellaW94** [AbsaOSS/ABRiS#324] [Q&A]
Hi community,
thanks for the high quality contribution and our team has been relying on abris all the time. However, we are on the migration path to Spark 3.3.0 from Spark 3.1.2. I've noticed that Abris is not supporting Spark 3.3 anymore, is there any known vulnerability by using it? Thanks

**Comment by cerveada** [comment]
Hello, we simply have not yet tested Abris on Spark 3.3. It may work fine, it might not. 

@kevinwallimann are you aware of anyone using Abris with Spark 3.3 ?

**Reply by kevinwallimann** [reply]
I haven't had time to test Abris on Spark 3.3. @EstellaW94 please don't be discouraged by the fact that Spark 3.3 is not mentioned on the supported versions matrix. It doesn't mean that it's not supported, we have just not tested yet.
